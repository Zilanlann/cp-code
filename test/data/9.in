711544965
747263453
