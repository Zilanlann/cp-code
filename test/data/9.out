1
531711547510664145
